### PR TITLE
[Darkside] Loader CSS update

### PR DESCRIPTION
**Pull request by KenAJoh**

### CODE REVIEW

**Comment by HalvorHaugan** [comment]
Still WIP?

**Comment by KenAJoh** [comment]
> Member

No, should be good-to-go

File: @navikt/core/css/darkside/loader.darkside.css [hunk 1]
```diff
@@ -5,88 +5,101 @@
...
-}
+  --__axc-loader-stroke-width: 5.9;
 
-.navds-loader__foreground {
-  animation: loader-dasharray 1.8s ease-in-out infinite;
-  stroke-dasharray: 80px, 200px;
-  stroke-dashoffset: 0;
-  stroke: var(--ac-loader-stroke, var(--a-border-default));
-}
+  &.navds-loader--3xlarge {
```
Comment (HalvorHaugan) [inline @navikt/core/css/darkside/loader.darkside.css:10]: Why nest this?

**Review by HalvorHaugan — DISMISSED** [review]

**Comment by HalvorHaugan** [comment]
I get some weird shaking in Chrome 😅

https://github.com/user-attachments/assets/7a9b1b57-a493-4777-b653-1898b7010a30

Only seems to happen when there are multiple loaders, so I don't think we need to worry about it.

**Comment by KenAJoh** [comment]
> I get some weird shaking in Chrome 😅

@HalvorHaugan Is this new in this PR, or the case with the old loader to?

File: @navikt/core/css/darkside/loader.darkside.css (hunk 1 above)
Comment (KenAJoh) [reply @navikt/core/css/darkside/loader.darkside.css:10]: Personally like to have changes in local tokens before it is used ` --__axc-loader-stroke-width`. Feels disorienting to have "definition -> use -> definition -> definition" in some cases. But for a short file like this it doesn't really matter

**Comment by HalvorHaugan** [comment]
Only with the new darkside css in this PR

**Comment by KenAJoh** [comment]
@HalvorHaugan So strange, no changes were made to how this were handled 🤔 Does it change if you remove `stroke-linecap: round;` in this CSS? That should be the only actual change
```
.navds-loader__foreground {
  animation: loader-dasharray 1.5s ease-in-out infinite;
  stroke-dasharray: 80px, 200px;
  stroke-dashoffset: 0;
  stroke: var(--ax-border-strong);
  stroke-linecap: round;
}
```

File: @navikt/core/css/darkside/loader.darkside.css (hunk 1 above)
Comment (HalvorHaugan) [reply @navikt/core/css/darkside/loader.darkside.css:10]: I think you misunderstand. I'm wondering why you put `&.navds-loader--3xlarge` etc. inside `.navds-loader {`?
Do you need the selector to be `.navds-loader.navds-loader--3xlarge`?

**Comment by HalvorHaugan** [comment]
@KenAJoh Yes, it doesn't happen if I remove `stroke-linecap` 😅 It doesn't happen on my personal computer though, so it's probably just something weird with my work computer 😅 I wonder what it could be though, because it happens in incognito as well as in Chrome Beta, Canary and Dev too 🙃

**Review by HalvorHaugan — APPROVED** [review]

**Pull request merged**